.github/workflows/pipeline.yml

```diff
@@ -45,6 +45,9 @@ jobs:
       - name: Check format
         run: npm run format:check
 
+      - name: Check autogenerated docs
+        run: npm run docs:gen && git diff --exit-code
```

Comment:
Docs checks are run in CI. Should we split each code check on its own job, so they are run in parallel?